torch/distributed/tensor/_op_schema.py

```diff
@@ -586,6 +564,7 @@ def _inplace_rewrap_schema_suggestion(self, origin_schema: "OpSchema") -> None:
                 new_arg_schema.append(arg)
         self.args_schema = tuple(new_arg_schema)
         self.kwargs_schema = origin_schema.kwargs_schema
+        self._recompute_comparison_key()
```

Comment:
hmm, how did you confirm these are the only 2 places to call recompute? ideally we don't have to call it anywhere but post_init but this makes me wonder

Reply:
>  suggest add self._recompute_comparison_key() at the end of _inplace_rewrap_schema_suggestion for now to be safe

Great suggestion. We are replying to a comment thread about the line that implements it. :)